client/src/UploadForm.tsx

```diff
@@ -129,11 +134,11 @@ class UploadForm extends React.Component<{}, { slug: string, type: string, data:
   };
 
   render = () => {
-    const { slug, previewUrl, message, isLoading } = this.state;
+    const { slug, previewUrl, previewSuccessful, message, isLoading } = this.state;
     return <Form onSubmit={this.handleSubmit} className="Form">
       <h3 className="d-flex justify-content-center">Add an icon</h3>
       <FileUpload
-        label="Upload an image file"
+        label="Upload an image file (Recommended no larger than 64x64px)"
```

Comment:
Maybe instead of adding text to the label, a `span` element with the `text-muted` class could be added below the form label in FileUpload so that it draws less attention. I've noticed the size issue starts occurring just over 9kB, so it could recommend a fixed sized limit eg. `<span className="text-muted"> (Maximum size: ≈9kB)</span>`.

![image](https://user-images.githubusercontent.com/20955511/193716217-3197b00d-04d5-4dbf-b2c6-a261a72133a5.png)

Let me know what you think